add.php

```diff
@@ -193,7 +193,7 @@
     $form = new XoopsThemeForm(_ADSLIGHT_ADD_LISTING, 'submitform', 'add.php');
```

Comment:
I think this should be
```php
$form = new XoopsThemeForm(_ADSLIGHT_ADD_LISTING, 'submitform', 'add.php', 'post', true);
```
If a XoopsForm is involved, it is best to keep all the logic in that form setup, and avoid using the Smarty tag option.